main.go

```diff
@@ -119,12 +122,16 @@ func (e Env) BugzillaDSN() string {
 		log.Infof("MYSQL_HOST overridden by local env: %s", val)
 		mysqlhost = val
 	} else {
-		mysqlhost = e.Udomain("auroradb")
+		mysqlhost = e.GetSecret("MYSQL_HOST")
+	}
+
+	if mysqlhost == "" {
+		log.Fatal("MYSQL_HOST is unset")
 	}
 
 	return fmt.Sprintf("%s:%s@tcp(%s:3306)/bugzilla?multiStatements=true&sql_mode=TRADITIONAL&timeout=5s&collation=utf8mb4_unicode_520_ci",
-		e.GetSecret("MYSQL_USER"),
-		e.GetSecret("MYSQL_PASSWORD"),
+		e.GetSecret("BUGZILLA_DB_USER"),
```

Comment:
I'm not sure what you are doing here. Are you trying to backport the changes in the fork uneet-ins/env here?

Since in the uneet-ins environment you have changed variable names and they are not updated in:

```
[hendry@t480s ~]$ ssm uneet-prod BUGZILLA_DB_PASSWORD
None
[hendry@t480s ~]$ ssm uneet-demo BUGZILLA_DB_PASSWORD
None
```

Reply:
> 'm not sure what you are doing here. Are you trying to backport the changes in the fork uneet-ins/env here?

Yes that's what I'm trying to do here

Reply:
Why? You have changed names and backporting these name changes will create work.

Reply:
> Since in the uneet-ins environment you have changed variable names and they are not updated in:
```
[hendry@t480s ~]$ ssm uneet-prod BUGZILLA_DB_PASSWORD
None
[hendry@t480s ~]$ ssm uneet-demo BUGZILLA_DB_PASSWORD
None
```
These have been updated in the PROD environment now.
Will do the DEMO is a few minutes

Reply:
The parameters BUGZILLA_DB_PASSWORD and BUGZILLA_DB_USER have been added to the parameters store for the DEMO and PROD environment now

Reply:
So is the idea now that uneet-ins projects use unee-t/env since now it's the same as Unee-T-INS /
env?

The reason why people use libraries and such is reduce code duplication. Have two repos doing the exact same thing creates confusion.

Reply:
> So is the idea now that uneet-ins projects use unee-t/env since now it's the same as Unee-T-INS/env?

Yes, this is exactly the idea

Reply:
Once we are comfortable that unee-t/env can work nicely with both Unee-T and any other Unee-T installation (including but not limited to Unee-T INS) then we will have no need for the Unee-T INS fork in the first place.

Ideally we can do the same for all dependencies (apienroll, invite, unit, lambda2sqs, etc...)